persistence/sql/persister_consent.go

```diff
@@ -153,18 +154,35 @@ func (p *Persister) GetConsentRequest(ctx context.Context, challenge string) (*c
 }
 
 func (p *Persister) CreateLoginRequest(ctx context.Context, req *consent.LoginRequest) error {
-	return errorsx.WithStack(p.Connection(ctx).Create(req))
+	f := flow.NewFlow(req)
+	return errorsx.WithStack(p.Connection(ctx).Create(f))
```

Comment:
We have a helper `github.com/ory/x/sqlcon.HandleError` which will properly wrap the different database errors and return a common error, e.g. for conflicts. Please use that one instead :)